incubator/solr/Chart.yaml

```
@@ -2,7 +2,7 @@
 
 apiVersion: "v1"
 name: "solr"
-version: "1.4.0"
+version: "1.4.1"
```

this is a new feature please bump
```suggestion
version: "1.5.0"
```